Ansible/roles/cloudstack-manager/tasks/setup-mgmt.yml

```diff
@@ -50,7 +50,11 @@
   tags:
     - setup_mgmt
 
-
+- name: Setup CloudStack Manager (SUSE)
+  shell: sleep {{ play_hosts.index(inventory_hostname) | int * 15 }} && /usr/bin/cloudstack-setup-management
+  when: ansible_distribution == "SUSE"
+  tags:
+    - setup_mgmt
```

Comment:
why do we need a separate task for these? a lot of them are only different in conditions and those can be combined.